docs/developers/release-process/pre-release-checklist.md

```diff
@@ -0,0 +1,66 @@
+# Ethrex Pre-Release Checklist (Draft)
+
+This checklist helps ensure a smooth pre-release process for ethrex. Before proceeding, replace placeholders like `X.Y.Z-rc.N` with the actual version number (e.g., `1.2.3-rc.1`). Mark each test as completed by checking the box (e.g., `- [x]`) or adding a ✅ or notes in the table cells. If a test doesn't apply, note "N/A" with a brief reason.
+
+Refer to the ethrex documentation for detailed setup instructions. All tests should be run on a clean environment to avoid interference from previous builds.
+
+## 1. Test the Binaries
+
+### Prerequisites
+
+Before testing, download the [pre-release binaries from the release artifacts](https://github.com/lambdaclass/ethrex/releases).
+
+### Running the Tests
+
+- **Snapsync Hoodi**: Follow the [snap sync guide for Hoodi](https://docs.ethrex.xyz/l1/running/index.html) to initialize and sync a test network.
```

Comment:
I think we can automate the syncing in the CI, which simplifies doing it for other architectures